### PR TITLE
[CDAP-1284] Added dataset metric tag to tag prefix part of aggregation r...

### PR DESCRIPTION
**Pull request by bdmogal**
Build: https://builds.cask.co/browse/CDAP-RBT115-1

### CODE REVIEW

File: cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MetricRecordsWrapper.java [hunk 1]
```diff
@@ -64,28 +64,38 @@
     rules.add(new Rule(ImmutableList.of(Constants.Metrics.Tag.CLUSTER_METRICS)));
     // app, prg type, prg name
     rules.add(new Rule(ImmutableList.of(Constants.Metrics.Tag.APP, Constants.Metrics.Tag.PROGRAM_TYPE,
-                                        Constants.Metrics.Tag.PROGRAM)));
+                                        Constants.Metrics.Tag.PROGRAM),
+                       Constants.Metrics.Tag.DATASET));
```
Comment (gokulavasan) [inline cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MetricRecordsWrapper.java]: Fix alignment

Comment (bdmogal) [inline cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MetricRecordsWrapper.java]: The alignment is correct. Line 68 is the second argument of the ``Rule`` constructor. It is not part of the ``ImmutableList``, which is the first argument to the ``Rule`` constructor.

Agreed that its hard to notice though :)

Comment (gokulavasan) [inline cdap-watchdog/src/main/java/co/cask/cdap/metrics/process/MetricRecordsWrapper.java]: Oh ok!

**Comment by bdmogal** [comment]
Closing PR. Needs a little more testing. Will open again.

**Pull request closed without merging**